gradle/libs.versions.toml

```diff
@@ -45,8 +45,8 @@ javalite = "3.25.5"
 jsonassert = "1.5.0"
 kotest = "5.9.0" # Do not use 5.9.1 because it reverts the fix for https://github.com/kotest/kotest/issues/3981
 kotestAssertionsCore = "5.8.1"
-kotlin = "1.8.22"
-ktorVersion = "2.3.2"
+kotlin = "2.0.21"
```

Comment:
FWIW Gradle 9 also uses Kotlin 2 :) 
(I hope this makes our transition smoother when the time comes)
https://docs.gradle.org/9.0.0-rc-1/release-notes.html#update-to-kotlin-2